internal/file/archive/tarball.go

```diff
@@ -5,38 +5,57 @@ import (
 	"bytes"
 	"compress/gzip"
 	"strings"
+	"time"
 )
 
+// Writer allows writing file entries into a tarball.
+type Writer struct {
+	tw      *tar.Writer
+	modTime time.Time
+}
+
+// New returns a Writer for writing tar file entries into the passed
+// tar.Writer. All the file headers will have a mod time equal to the
+// time New() was called.
+func New(tw *tar.Writer) *Writer {
+	return &Writer{
+		tw:      tw,
+		modTime: time.Now(),
+	}
```

Comment:
I'm curious about why we don't directly set the `tar.Header.ModTime` to `time.Now` vs declaring a new struct ? Especially since we don't allow the time to be set in the initialization of the struct.

Reply:
This way, all file entries have the same time, which is roughly the moment the bundle was created... I guess it doesn't matter much either way. I'll simplify it if you feel strongly about this 😃